experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -193,19 +198,27 @@ export class NodeSDK {
         { [SemanticResourceAttributes.SERVICE_NAME]: this._serviceName }
       ));
 
-    if (this._tracerProviderConfig) {
-      const tracerProvider = new NodeTracerProvider({
-        ...this._tracerProviderConfig.tracerConfig,
+    if (this._tracerProviderConfig || this._useEnvExporters) {
```

Comment:
This is always `true`, right? :thinking: 
Looking at the constructor, either `counfigureTracerProvider()` creates a config, or `_useEnvExporters` is set to `true`.